codice/ddf
# DDF-1897 - Metacard Versioning

#### What does this PR do?
Adds Support for Metacard Versioning
#### Who is reviewing it?
@jlcsmith @jrnorth @stustison @pklinef @kcwire @bdeining and if a fireman wants to.. @coyotesqrl 
#### How should this be tested?
Build and install, Enable Metacard Versioning with `feature:install catalog-versioning-plugin`, ingest a record, update a record, delete a record, confirm there is a history metacard created for each action.
#### Any background context you want to provide?
Support added by use of a `PostIngestPlugin` and a new type of metacard (`HistoryMetacardImpl`)
#### What are the relevant tickets?
[DDF-1897](https://codice.atlassian.net/browse/DDF-1897)
#### Checklist:
